<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>cook_window</class>
 <widget class="QDialog" name="cook_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>777</width>
    <height>463</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="QTableView" name="unconfirmed"/>
   </item>
   <item row="3" column="0" colspan="3">
    <widget class="QTableView" name="confirmed"/>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Поступившие заказы</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Повар</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="cookName">
         <property name="text">
          <string>cook_name</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="takeOrder">
         <property name="text">
          <string>Принять заказ</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton">
         <property name="text">
          <string>Отменить заказ</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="readyOrder">
         <property name="text">
          <string>Сообщить о готовности</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Текущие заказы:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QTableView" name="tableChat"/>
   </item>
   <item row="0" column="1">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Чат</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QPushButton" name="chatSend">
     <property name="text">
      <string>Отправить</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="chatInput"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
